src/lib/braze.ts

```diff
@@ -0,0 +1,23 @@
+import { data as sd } from "sharify"
+
+export const setup = async () => {
```

Comment:
#minor a little more explicit naming would be nice `setupBraze`, that way when it's imported and called in the consuming file the future reader knows exactly whats happening.